src/Apps/__stories__/Collect.story.tsx

```diff
@@ -5,7 +5,17 @@ import { collectRoutes } from "../Collect2/collectRoutes"
 
 storiesOf("Apps/Collect", module)
   .add("Collect", () => {
-    return <MockRouter routes={collectRoutes} initialRoute="/collect" />
+    return (
+      <MockRouter
+        routes={collectRoutes}
+        initialRoute="/collect"
+        context={{
+          // FIXME: Remove after A/B test completes
+          // @ts-ignore
+          COLLECTION_HUBS: "experiment",
```

Comment:
When working in Storybooks, we need to inject the ENV var in similar to how we do pass it into context on the Force side of things via A/B test